lang/po/Firmware_cs.po

```diff
@@ -2216,13 +2250,13 @@ msgstr "Vysouvam filament"
 #: ../../Firmware/mmu2_progress_converter.cpp:11
 #: ../../Firmware/mmu2_progress_converter.cpp:37
 msgid "Unloading to FINDA"
-msgstr ""
+msgstr "Vysunuti k FINDA"
 
 #. MSG_PROGRESS_UNLOAD_PULLEY c=20
 #: ../../Firmware/mmu2_progress_converter.cpp:12
 #: ../../Firmware/mmu2_progress_converter.cpp:38
 msgid "Unloading to pulley"
-msgstr ""
+msgstr "Vysunuti k idleru"
```

Comment:
@DRracer again copy/paste issue

Reply:
should read "Vysunuti ke kladce" if we keep the Pulley->Kladka translation